Cirrious/Cirrious.MvvmCross.Touch/Views/MvxTouchViewsContainer.cs

```diff
@@ -39,6 +41,14 @@ public virtual IMvxTouchView CreateView(MvxViewModelRequest request)
 
         protected virtual IMvxTouchView CreateViewOfType(Type viewType, MvxViewModelRequest request)
         {
+            var storyboardAttribute = viewType.GetCustomAttribute<MvxFromStoryboardAttribute>();
+            if (storyboardAttribute != null) 
+            {
+                var storyboard = UIStoryboard.FromName(storyboardAttribute.StoryboardName ?? viewType.Name, null);
+                var viewController = storyboard.InstantiateViewController(viewType.Name);
+                return (IMvxTouchView) viewController;
```

Comment:
if `storyboard` or `viewController` is null then should we throw an exception here - this won't help at runtime - but it would help any developer who is trying to debug why their view doesn't load?